sources/packages/web/src/services/http/ApplicationApi.ts

```diff
@@ -110,16 +110,18 @@ export class ApplicationApi extends HttpBaseClient {
     applicationId: number,
     payload: SaveStudentApplicationDto,
   ): Promise<void> {
-    try {
-      return await this.apiClient.patch(
+    await this.apiClient
```

Comment:
Understand that we did not use patchCall for a reason here. may be a comment would help otherwise during code copy paste it may get carried over to scenarios where we don't need it.